Source/Scene/Model.js

```diff
@@ -1454,6 +1454,191 @@ define([
+                    var technique = techniques[techniqueId];
+                    
+                    var programId = technique.program;
+                    if (programId === programName) {
+                        var primitiveAttributes = primitive.attributes;
+                        var attributeVarName;
+                        for (var attributeSemantic in primitiveAttributes) {
+                            if (primitiveAttributes.hasOwnProperty(attributeSemantic)) {
+                                attributeVarName = undefined;
+                                for (var attributeName in technique.parameters) {
+                                    if (technique.parameters.hasOwnProperty(attributeName)) {
+                                        if (defined(attributeVarName)) {
+                                            break;
+                                        }
+                                        if (technique.parameters[attributeName].semantic === attributeSemantic) {
+                                            for (var varName in technique.attributes) {
+                                                if (technique.attributes.hasOwnProperty(varName)) {
+                                                    if (technique.attributes[varName] === attributeName) {
+                                                        attributeVarName = varName;
+                                                        break;
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                                // Skip if the attribute is not used by the material, e.g., because the asset was exported
+                                // with an attribute that wasn't used and the asset wasn't optimized.
+                                if (defined(attributeVarName)) {
+                                    var attributeId = primitiveAttributes[attributeSemantic];
+                                    var a = accessors[attributeId];
+                                    if (a.hasOwnProperty('extensions') && processedAccessors.indexOf(attributeId) < 0) {
+                                        var extensions = a.extensions;
+                                        if (extensions.hasOwnProperty('WEB3D_quantized_attributes')) {
+                                            // Only one of each semantic type can be decoded per shader
+                                            if (decodedAttributes.indexOf(attributeSemantic) >= 0) {
+                                                // Assign any unresolved primitives to a new program
+                                                if (!defined(unresolvedProgramId)) {
+                                                    unresolvedProgramId = programName + '_r';
```

Comment:
Not a huge deal, but I could imagine `_r` more easily hitting a collision with a real name.  Although it could also happen with `czm_`, we generally treat it as a reserved namespace.  You could even do `czm_resolved` if you prefer.